GuessingGame.html

```diff
@@ -1,60 +1,84 @@
+<html>
+<head>
+<title>Bad Ass Ladies of the Supreme Court</title> </head>
+<body>
+<h1>Bad Ass <strong>Ladies</strong> of The United States
+<strong>Supreme Court</strong></font></h1>
```

Comment:
I'd also like to see some more use of block elements, such as <Header>, <Main>, and <Footer>. Those will help create some segregation in your page and will give you the flexibility later on to manipulate your page with both CSS and JS.